canonical-websites/jp.ubuntu.com
# Vanilla v1

## Done
Updated the codebase to support Vanilla v1+. Add the new global navigation module.

## QA
- Pull a branch
- Run `./run`
- Open up http://0.0.0.0:8012/
- Check the page looks good on all view ports
- Check the size of the page has shrunk from the live site
- Check that there are no accessibility errors when testing with AA
- Check that the markup is valid

## Issue / Card
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/9 https://github.com/ubuntudesign/vanilla-squad/issues/10 https://github.com/canonical-websites/jp.ubuntu.com/issues/20 https://github.com/canonical-websites/jp.ubuntu.com/issues/18 https://github.com/canonical-websites/jp.ubuntu.com/issues/16 https://github.com/canonical-websites/jp.ubuntu.com/issues/15

## Screenshots
![ubuntu 1](https://user-images.githubusercontent.com/1413534/27993768-0336d352-64a8-11e7-8948-0f199cc1e19c.png)

## Demo
http://jp.ubuntu.com-vanilla-v1.demo.haus
